src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -108,4 +122,116 @@ public void exec(final URI home) throws IOException {
             }
         );
     }
+    
+    /**
+     * Try to send 2 requests in 1 tcp connection.
+     * @throws IOException
+     */
+    @Test
+    public void persistenceConnection() throws IOException {
```

Comment:
@hs3180 I see only one request in this test. How do you check that's the connection not closed after first request?